data/phones/HOWTO.md

```diff
@@ -54,7 +54,9 @@ the [fork and pull](../../CONTRIBUTING.md) model for contributions.
     using `git add`. The `.phones` file must use the [NFC Unicode 
     normalization](https://en.wikipedia.org/wiki/Unicode_equivalence#Normalization).
     If you used `../src/list_phones.py` to create the `.phones` file, then it
-    should be in this form already.
+    should be in this form already. Otherwise, in `../src/`, you can run
+    `python normalize.py <your-file> NFC` to put your file in the correct
```

Comment:
This should be `./normalize.py`, since all the scripts in this directory should be executable now.

Sidebar: I just made `normalize.py` executable; my fault for not catching that earlier. As is style elsewhere here, please make all of the data scripts executable as follows:

1. on the very first line add `#!/usr/bin/env python`
2. run `chmod +x yourscript.py`

Then you can do `./yourscript.py`.

Reply:
Alright, it's taken care of!